cmd/podman/machine/set.go

```diff
@@ -53,5 +89,22 @@ func setMachine(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	return vm.Set(vmName, setOpts)
+	if cmd.Flags().Changed("rootful") {
+		setOpts.Rootful = &setFlags.Rootful
+	}
+	if cmd.Flags().Changed("cpus") {
+		setOpts.CPUs = &setFlags.CPUs
+	}
+	if cmd.Flags().Changed("memory") {
+		setOpts.Memory = &setFlags.Memory
+	}
+	if cmd.Flags().Changed("disk-size") {
+		setOpts.DiskSize = &setFlags.DiskSize
+	}
+
+	setErrs, err := vm.Set(vmName, setOpts)
+	for _, err := range setErrs {
+		fmt.Fprintf(os.Stderr, "%v\n", err)
+	}
```

Comment:
I think we already have a package for this
https://github.com/containers/podman/blob/main/cmd/podman/utils/error.go

Reply:
Looks like that type is in `cmd/podman`, and I don't want to import `cmd/podman` into `pkg/machine`